changes.d/fix.5924.md

```diff
@@ -0,0 +1 @@
+Ensure that graphs with dependency offsets on the right hand side are flagged to users.
```

Comment:
I think this needs to be rephrased. What is "flagged to users"?  And RHS offsets are only illegal if the the does not appear elsewhere with an offset. 

Expressing this concisely ain't exactly easy though. How about this:

```suggestion
Validation: a cycle offset can only appear on the right of a dependency if the task's cycling is defined elsewhere with n offset.
```
(Ideally we might say the `the task's cycling must `.